repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json

```diff
@@ -1,22 +1,22 @@
 {
-  "name": "ReposeFakeKeystonev2",
-  "version": "0.1.0",
-  "description": "Mock Keystone Service",
-  "readme": "README.md",
+  "name": "service-mocks",
+  "version": "1.0.0",
```

Comment:
Should we bump this to a full major version release or are we only at a minor?
This feels more like a minor.